pkg/broker/ensure.go

```diff
@@ -34,7 +34,7 @@ import (
 	"k8s.io/client-go/rest"
 )
 
-func Ensure(config *rest.Config) error {
```

Comment:
Why is the crds bool added, where is it used? :-?

Reply:
Oops, something got lost in a rebase it seems...

Reply:
Now restored.